<!DOCTYPE html>
<html lang="en"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Reseach Projects | Marzieh Haghighi</title>
<meta name="generator" content="Jekyll v4.3.2" />
<meta property="og:title" content="Reseach Projects" />
<meta name="author" content="Marzieh Haghighi" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Project page." />
<meta property="og:description" content="Project page." />
<link rel="canonical" href="http://localhost:4000/about.html" />
<meta property="og:url" content="http://localhost:4000/about.html" />
<meta property="og:site_name" content="Marzieh Haghighi" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Reseach Projects" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","author":{"@type":"Person","name":"Marzieh Haghighi"},"description":"Project page.","headline":"Reseach Projects","url":"http://localhost:4000/about.html"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/css/style.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Marzieh Haghighi" />
</head>
<body><header class="site-header">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Marzieh Haghighi</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span>
        </label>

        <div class="trigger"><a class="page-link" href="/about.html">Reseach Projects</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <article class="post">

  <header class="post-header">
    <h1 class="post-title">Reseach Projects</h1>
  </header>

  <div class="post-content">
    <h3 id="recent-projects">Recent Projects:</h3>

<hr />
<h3 id="identifying-and-targeting-signatures-of-psychiatric-diseases">Identifying and targeting signatures of psychiatric diseases:</h3>

<div style="overflow: auto">
  <div style="float: right; margin-left: 10px">
    <img src="assets/img/mito_p.png" alt="Alt Text" width="200" />
  </div>
    We developed a high-throughput assay system for identifying candidate small molecules for psychiatric disease. Mitochondrial abnormalities in patients with psychiatric illness have been reported in the literature through genetic, brain imaging, and peripheral cell studies. Using high-content microscopy imaging and machine learning, we identified skin fibroblasts cellular fingerprints of disease based on structural changes in mitochondria among patients with four different psychiatric conditions: schizophrenia, schizoaffective disorder, bipolar disorder, and major depression. Taking the differential mitochondrial structure fingerprints among the groups of patients, we searched for possible therapeutics by mining existing image datasets of cell samples treated with tens of thousands of small molecules. We are currently collaborating with NIH NCATS to run experiments to validate this assay system and then screen to identify novel candidate compounds targeting these diseases.
</div>

<p><em>(In preparation for submission)</em></p>

<hr />
<h3 id="protein-localization-of-rare-disease-variants">Protein localization of rare disease variants:</h3>

<h4 id="systematic-characterization">Systematic characterization</h4>

<div style="overflow: auto">
  <div style="float: right; margin-left: 10px">
    <img src="assets/img/rare_p.png" alt="Alt Text" width="200" />
  </div>
  <!-- <div style="text-align: justify"> -->
In collaboration with Taipale lab  (University of Toronto), we systematically characterize the protein localization of disease variants by high-content microscopy. The Taipale lab had generated a catalog of types of mislocalization, scoring images by eye. However, we improved the characterization of the alleles with an advanced, automated morphological profiling approach. We tapped into the potential of using the entire spectrum of morphological measurements for detecting mutant phenotypes, instead of focusing on localization alone.
<!-- </div> -->
</div>

<ul>
  <li><em>Link to the github repo</em></li>
  <li><em>Link to the paper</em></li>
</ul>

<h4 id="the-path-for-drug-discovery">The path for drug discovery</h4>

<!-- <div style="text-align: justify"> -->
<p>The path for a virtual drug screen in this project is not straightforward. In this project, we donâ€™t have access to a compound dataset of protein localizations to use as a reference for a virtual drug screen, and carrying out hundreds of individual drug screens is impractical due to the cost. We are facilitating a huge screen of compounds, however, by taking advantage of the pooled optical profiling strategy developed by Blainey lab at the Broad Institute. By pooled optical screens, we can test thousands of drugs against hundreds of barcoded disease-associated phenotypes in a single reasonably sized experiment, which is currently in the pilot stage.
<!-- </div> --></p>

<p><em>(In progress)</em></p>

<hr />

<h3 id="decoding-barcodes-from-in-situ-sequencing-images">Decoding barcodes from In-Situ-Sequencing images</h3>
<div style="overflow: auto">
  <div style="float: right; margin-left: 10px">
    <img src="assets/img/plepi_p.png" alt="Alt Text" width="300" />
  </div>
  <!-- <div style="text-align: justify"> -->
We developed and introduced a framework which exploits a novel self-training learning strategy to address the barcode calling problem. The proposed self-training strategy can start from a small set of noisy labels and correct for noise overfitting by batch supervision. This work involved creative incorporation of a side prior knowledge about barcodes which is not in the regular form of annotations. Despite the focus of this work on the barcode calling problem, the introduced general learning strategy can be extended to use in other domains of data with contextual information. And crucially, this work will contribute to the efficiency and quality of the pooled drug screen experiments and data analysis.
  <!-- </div> -->
</div>

<ul>
  <li><em>Link to the github repo</em></li>
  <li><em>Link to the paper</em></li>
</ul>

<hr />
<h3 id="multi-modal-data-regimen-coupled-with-cell-painting">Multi-modal data regimen coupled with Cell Painting</h3>
<h4 id="circle-commentary-on-the-opportunities-and-benchmarking-the-problem-at-the-population-level">$\circle$ Commentary on the opportunities and benchmarking the problem at the population-level</h4>
<div style="overflow: auto">
  <div style="float: right; margin-left: 10px">
    <img src="assets/img/ros_p.png" alt="Alt Text" width="150" />
  </div>
  <!-- <div style="text-align: justify"> -->
  We delve into the potential opportunities that a multi-modal gene expression and cell painting dataset provide. We define biological problems that use the shared and complementary information in these two data modalities, provide baseline analysis and evaluation metrics for multi-omic applications by using four genetic and compound datasets to test hypotheses of complementary or predictive nature of these two data types at the bulk level. 
  <!-- </div> -->
</div>

<ul>
  <li><em>Link to the github repo</em></li>
  <li><em>Link to the paper</em></li>
</ul>

<h4 id="multi-domain-representation-learning">Multi-domain representation learning</h4>

<div style="overflow: auto">
  <div style="float: right; margin-left: 10px">
    <img src="assets/img/rep_p.png" alt="Alt Text" width="300" />
  </div>
   Learning representations directly from single cell images has received attention in recent years. Unbiased approaches suffer from noise and redundancy which prevent us from exploiting the full power of such data sources in morphological profiling and their emerging application in the drug discovery pipeline. Ideally, we want to learn representations that are unbiased from weak and erroneous human based annotations and at the same time disentangled from technical variations and experimental noise. We propose a novel multi-purpose framework (CellStyle) for learning representations directly from single cell images using various types of supervisory signals without compromising any unknown yet potentially useful properties of the data. The framework is based on an autoencoder that directly learns two orthogonal subspaces of data, called content and style. We define style and content in the context of various profiling tasks each using a unique source of supervisory signal. 
</div>

<p><em>(In progress)</em></p>

<hr />

<h3 id="past-projects">Past Projects:</h3>
<hr />
<h3 id="automatic-renal-segmenter">Automatic Renal Segmenter</h3>
<div style="overflow: auto">
  <div style="float: right; margin-left: 10px">
    <img src="assets/img/kidney_p.png" alt="Alt Text" width="300" />
  </div>
  Kidney function evaluation using dynamic contrast-enhanced MRI (DCE-MRI) images could help in diagnosis and treatment of kidney diseases of children. Automatic segmentation of renal parenchyma is an important step in this process. In this paper, we propose a time and memory efficient fully automated segmentation method which achieves high segmentation accuracy with running time in the order of seconds in both normal kidneys and kidneys with hydronephrosis. The proposed method is based on a cascaded application of two 3D convolutional neural networks that employs spatial and temporal information at the same time in order to learn the tasks of localization and segmentation of kidneys, respectively. 
</div>

<ul>
  <li><em>Link to the paper</em></li>
</ul>

<hr />

<h3 id="eeg-assisted-modulation-of-sound-sources-in-the-auditory-scene">EEG-assisted Modulation of Sound Sources in the Auditory Scene</h3>
<div style="overflow: auto">
  <div style="float: right; margin-left: 10px">
    <img src="assets/img/thesis_p.png" alt="Alt Text" width="500" />
  </div>
  <!-- <div style="text-align: justify"> -->
    Noninvasive auditory attention detection Brain Computer Interfaces (BCIs) could be useful for improved hearing aids in the future. EEG (electroencephalography) has been demonstrated to exhibit useful evidence regarding auditory attention in the presence of multiple competing speech waveforms. This work is a novel attempt to investigate the feasibility of online modulation of sound sources by probabilistic detection of auditory attention, using a noninvasive EEG-based brain computer interface. Proposed online system modulates the upcoming sound sources through gain adaptation which employs probabilistic decisions (soft decisions) from a classifier trained on offline calibration data. The goal of the online system is using the probabilistic information of the user's attention to enhance the concentration of the user on the target source in multi-speaker scenarios. Through 3 phases of study and experiments we have explored and improved the system by optimizing the model, features and the experimental designs. 
  <!-- </div> -->
</div>

<ul>
  <li><em>Links (1,2) to the papers</em></li>
</ul>

<hr />

  </div>

</article>
      </div>
    </main><footer class="site-footer h-card">
  <data class="u-url" href="/"></data>

  <div class="wrapper">

    <div class="footer-col-wrapper">
      <div class="footer-col">
        <div class="contact-links">
          <p class="contact-me">
            <a href="mailto:marzieh.haghighi@gmail.com">
            <svg class="svg-icon orange">
              <use xlink:href="/assets/minima-social-icons.svg#rss"></use>
            </svg><span>Contact</span>
          </a>
        </p>
      </div>
        <ul class="contact-list">
          <li class="p-name">Marzieh Haghighi</li>
          
        </ul>
      </div>
      <div class="footer-col">
        <div class="social-links"><ul class="social-media-list"><li>
  <a rel="me" href="https://github.com/MarziehHaghighi" target="_blank" title="github">
    <svg class="svg-icon grey">
      <use xlink:href="/assets/minima-social-icons.svg#github"></use>
    </svg>
  </a>
</li>
<li>
  <a rel="me" href="https://scholar.google.com/citations?user=E-v_EoEAAAAJ&hl=en" target="_blank" title="google_scholar">
    <svg class="svg-icon grey">
      <use xlink:href="/assets/minima-social-icons.svg#google_scholar"></use>
    </svg>
  </a>
</li>
<li>
  <a rel="me" href="https://www.linkedin.com/in/marzieh-haghighi/" target="_blank" title="linkedin">
    <svg class="svg-icon grey">
      <use xlink:href="/assets/minima-social-icons.svg#linkedin"></use>
    </svg>
  </a>
</li>
</ul>
</div>
      </div>
    </div>

  </div>

</footer>
</body>

</html>
